code_to_gast/helpers/py_helpers.py

```diff
@@ -176,6 +176,47 @@ def name_to_gast(node):
 def unary_op_to_gast(node):
     return {"type": "unaryOp", "op": pyop_to_str(node.op), "arg": pr.node_to_gast(node.operand)}
 
+"""
+takes argument from function declaration and makes into gast node
+"""
+def arg_to_gast(node):
+    return pr.node_to_gast(node.arg)
+
+"""
+Takes native string class from python and creates gast node
+This node is used to store function names and arguments which
+is why the node type is name for identifier
+"""
+def str_to_gast(node):
```

Comment:
This is slightly confusing to me since we already have ways of converting strings to gast. I don't think I'm quite understanding what you're doing here so maybe add some comments